### ISSUE TITLE
Port Forwarding on Server

### ISSUE THREAD

**Issue by gvsurenderreddy** [jpillora/chisel#165]
Hello sir 
Port mapping    possible using Chisel on client to server. 
 Example server port 22 will be mapped to the local port 8787 Thanks again sir

**Comment by patrick-othmer** [comment]
Sure

Local port on chisel server will be 8787
```
chisel client your.chisel.server R:8787:localhost:22
```

**Comment by gvsurenderreddy** [comment]
Port 8787 should be free on Server My problem is i want connect with localhost 22

**Comment by ezbik** [comment]
>If Server has SSH running on Port 22 then Port mapping is not possible

Of course! How can 2 apps  listen on the same IP:port combination?? Bind Chisel to 8787  (i.e. not 22) and use iptables DNAT for some specific clients, so they   connect to `chisel_server:22` and actually go to `chisel_server:8787` which is `chisel_client:22`
